src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -337,6 +337,13 @@ public function get_items( $request ) {
 			}
 		}
 
+		if (
+			isset( $registered['search_semantics'], $request['search_semantics'] )
```

Comment:
We don't technically need the `isset` check since the rest request object will just return `null` in this case.

Reply:
Fair. I feel like it's okay keeping though for consistency. Maybe also a tiny bit safer.

Reply:
Works for me.